GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -79,17 +79,20 @@ public override IEnumerable<EventListener> CreateTelemetryListeners(string provi
         {
             string gitBinRoot = this.GitInstallation.GetInstalledGitBinPath();
 
-            var etwListener = ETWTelemetryEventListener.CreateIfEnabled(gitBinRoot, providerName, enlistmentId, mountId);
+            ETWTelemetryEventListener etwListener = ETWTelemetryEventListener.CreateIfEnabled(gitBinRoot, providerName, enlistmentId, mountId);
             if (etwListener != null)
             {
                 yield return etwListener;
             }
 
-            var daemonListener = TelemetryDaemonEventListener.CreateIfEnabled(gitBinRoot, providerName, enlistmentId, mountId, pipeName: "vfs");
-            if (daemonListener != null)
-            {
-                yield return daemonListener;
-            }
+            // TODO: enable the daemon-based telemetry listener once we're happy.
+            // See GitHub issue: https://github.com/Microsoft/VFSForGit/issues/739
+            //
+            // TelemetryDaemonEventListener daemonListener = TelemetryDaemonEventListener.CreateIfEnabled(gitBinRoot, providerName, enlistmentId, mountId, pipeName: "vfs");
+            // if (daemonListener != null)
+            // {
+            //     yield return daemonListener;
+            // }

```

Comment:
Just curious if StyleCop complained about the `//` without a space and if you had to add the space to each line. I think it does the last time I remember and what I usually do is add `////` by using Ctrl+E, C twice.

Reply:
Yes, StyleCop doens't like `//foo`, but wants `// foo`.